icons/funding.svg

```diff
@@ -0,0 +1,2 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M16.2695 11.5028C18.8922 11.5028 21.0209 9.3742 21.0209 6.75142C21.0209 4.12863 18.8922 2 16.2695 2C13.6467 2 11.518 4.12863 11.518 6.75142C11.518 9.3742 13.6467 11.5028 16.2695 11.5028ZM21.4485 16.8244C21.0779 16.4443 20.6122 16.2542 20.0706 16.2542H13.4186L11.442 15.5605L11.7556 14.6673L13.4186 15.304H16.0794C16.412 15.304 16.6781 15.1709 16.8966 14.9524C17.1152 14.7338 17.2197 14.4677 17.2197 14.1731C17.2197 13.66 16.9727 13.3084 16.4785 13.1088L9.56996 10.5525H7.7169V19.1051L14.3689 21.0057L21.9997 18.1548C22.0092 17.6512 21.8191 17.2045 21.4485 16.8244ZM5.81634 10.5525H2V21.0057H5.81634V10.5525Z" fill="#FDD835"/></svg>
```

Comment:
I suggest putting the official **GitHub Sponsors** icon (taken from https://primer.style/foundations/icons/heart-fill-16)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path fill="#db61a2" d="M7.655 14.916v-.001h-.002l-.006-.003-.018-.01a22.066 22.066 0 0 1-3.744-2.584C2.045 10.731 0 8.35 0 5.5 0 2.836 2.086 1 4.25 1 5.797 1 7.153 1.802 8 3.02 8.847 1.802 10.203 1 11.75 1 13.914 1 16 2.836 16 5.5c0 2.85-2.044 5.231-3.886 6.818a22.094 22.094 0 0 1-3.433 2.414 7.152 7.152 0 0 1-.31.17l-.018.01-.008.004a.75.75 0 0 1-.69 0Z"/></svg>
```

Reply:
@PKief